src/EA.Weee.RequestHandlers.Tests.DataAccess/Admin/Reports/GetProducerEeeDataHistoryCsvHandlerTests.cs

```diff
@@ -48,6 +48,7 @@ public async Task CreateResults_EeeHistoryCsvResult_PopulatedWithCorrectValues()
                 helper.CreateEeeOutputAmount(dataReturnVersion, producer1.RegisteredProducer, "B2C", 12, (decimal)1.12);
                 helper.CreateEeeOutputAmount(dataReturnVersion, producer1.RegisteredProducer, "B2C", 13, (decimal)1.13);
                 helper.CreateEeeOutputAmount(dataReturnVersion, producer1.RegisteredProducer, "B2C", 14, (decimal)1.14);
+                helper.CreateEeeOutputAmount(dataReturnVersion, producer1.RegisteredProducer, "B2C", 15, (decimal)1.15);
```

Comment:
I'd be tempted to populate these things in a loop to reduce maintenance burden -- although the assertions afterwards can't be similarly simplified except with reflection (unfortunately the code wasn't designed for the number of categories to be flexible)